Snakefile

```diff
@@ -0,0 +1,496 @@
+from datetime import datetime
+from snakemake.remote.GS import RemoteProvider as GSRemoteProvider
+GS = GSRemoteProvider()
+
+# --- Settings --- #
+# Current date in YYYY-MM-DD format:
+timeStamp = datetime.now().strftime("%Y-%m-%d")
+
+# Configuration is read from the config yaml:
+configfile: 'configuration.yaml'
+logFile = f"{config['global']['logDir']}/evidence_parser.{timeStamp}.log"
+
+# --- All rules --- #
+rule all:
+    input:
+        GS.remote(f"{config['ClinGen']['outputBucket']}/ClinGen-{timeStamp}.json.gz"),
+        GS.remote(f"{config['PheWAS']['outputBucket']}/phewas_catalog-{timeStamp}.json.gz"),
+        GS.remote(f"{config['SLAPEnrich']['outputBucket']}/slapenrich-{timeStamp}.json.gz"),
+        GS.remote(f"{config['Gene2Phenotype']['outputBucket']}/gene2phenotype-{timeStamp}.json.gz"),
+        GS.remote(f"{config['CRISPR']['outputBucket']}/crispr-{timeStamp}.json.gz"),
+        GS.remote(f"{config['SysBio']['outputBucket']}/sysbio-{timeStamp}.json.gz"),
+        GS.remote(f"{config['Phenodigm']['outputBucket']}/phenodigm-{timeStamp}.json.gz")
+
+# --- Auxiliary Rules --- #
+## help               : prints help comments for Snakefile
+rule help:
+    input: "Snakefile"
```

Comment:
This makes the pipeline run bound to the script directory. I think we should be more flexible than that. However this is not a blocker for merging.

Reply:
Added to the follow-up ticket.